.github/workflows/services_gtfs_ckan_uploader.yml

```diff
@@ -0,0 +1,31 @@
+name: Upload GTFS Schedules to CKAN
```

Comment:
Why are we doing this as a GH action. Should be a DAG

Reply:
Hey, this was me testing to see if it would even run on gh actions. It's unlikely the pipeline cluster as currently provisioned can support the upload (each node has < 3 gb ram). If we want the pipeline to run it, we will likely need to set aside a good chunk of time for someone to use the Kube Pod operator to run it on a bigger node, etc..